src/components/layout/header/language-switch.tsx

```diff
@@ -25,15 +25,18 @@ const StyledSelection = styled.select`
 
   color: inherit;
   padding-left: 18px;
+  text-align: right;
+  /*"direction: rlt" aligns the text right in Safari*/
+  direction: rtl;
 
   appearance: none;
   cursor: pointer;
 `;
 
-export const StyledChevron = styled(Icon)`
+export const StyledChevron = styled(Icon)<{ isEnglish: boolean }>`
   cursor: pointer;
-  /*margin-right: -22px makes the Chevron clickable*/
-  margin-right: -22px;
+  /*margin-right: negative margin makes the Chevron clickable*/
+  margin-right: ${(props) => (props.isEnglish ? '-26px' : '-22px')};
```

Comment:
so many magic numbers and in addition it's a different number per language 😳 Is that such a big issue here or what's happening?

Reply:
I don't think this line is ideal, but I couldn't think of a better solution. Maybe you have an idea.
The problem is that the `<select>` item has the width of the widest option. Without this distinction the distance between the chevron and the text would change depending on the language because of the `text-align: right` (s. screenshot)
<img width="80" alt="Bildschirmfoto 2022-03-10 um 08 01 11" src="https://user-images.githubusercontent.com/57712895/157606869-ae51a577-9eaa-4192-bef5-2d1c778cba68.png">
<img width="87" alt="Bildschirmfoto 2022-03-10 um 08 01 21" src="https://user-images.githubusercontent.com/57712895/157606873-c1707d22-e784-45e1-a5e5-ac022f998bec.png">